cmd/podman/networks/create.go

```diff
@@ -27,6 +29,9 @@ var (
 
 var (
 	networkCreateOptions entities.NetworkCreateOptions
+	opts                 = struct {
+		Opts []string
+	}{}
```

Comment:
Why is this a struct? A normal slice should be enough.

Reply:
copy/paste only (it used to have "Labels" too)